### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by hangon666** [changhongjian/Deep3DFaceReconstruction-pytorch#4]
Hi, thanks for your great job, byt the way, did you reimplement the loss functions and could you give me some guide?

**Comment by ForrestPi** [comment]
waiting for the training codes
